templates/scripts/setup-user-access.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+#
+# This script is to help user to create local AWS and Kubernetes configurations
+# 
+# You should have the following information ready before running:
+# 1. your AWS user name
+# 2. your AWS access key
+# 3. your user role
+
+IAM_USERNAME=$1
+ROLE=$2
+ENVIRONMENT=$3
+
+# templated variables
+AWS_ACCOUNT_ID=<% index .Params `accountId` %>
+PROJECT=<% .Name %>
+REGION=<% index .Params `region` %>
+
+# common functions
+function usage() {
+  echo "  Usage:"
+  echo "    $0 <iam_username> <role> <environment>"
```

Comment:
Could you put some more info in here similar to the comment at the top?

Reply:
sure